recipes/jupyter-rfb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jupyter-rfb" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_rfb-{{ version }}.tar.gz
+  sha256: 4c2cbab6b5c81379f30ccad3906c875c4d59d4758a36bb75805fb920d5b0bcf8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging >=0.7.9,<0.8
+    - jupyterlab >=3.0,<4
+    - pip
+    - python >=3.6
+  run:
+    - ipywidgets >=7.6.0
+    - numpy
+    - python >=3.6
+
+test:
+  imports:
+    - jupyter_rfb
+  commands:
+    - pip check
```

Comment:
meanwhile, it probably _is_ worth depending on `jupyterlab` for the test phase, and doing some "smoke" tests with the [labextension listing](https://github.com/search?q=org%3Aconda-forge+labextension+list&type=code) mechanims to verify your assets get deployed to `$PREFIX/share` and/or `$PREFIX/etc` and the metadata (e.g. `install.json`) is right.

Reply:
Yes, of course. Good point. I'll copy over some of those examples.